packages/libs/error-stack/tests/common.rs

```diff
@@ -1,9 +1,5 @@
 #![allow(dead_code)]
 
-pub fn create_report() -> Report<RootError> {
-    Report::new(RootError)
-}
-
```

Comment:
Can we keep this at the top, so the output will not change at any test change?

Reply:
fixed in [4c67564](https://github.com/hashintel/hash/pull/1290/commits/4c675645442c693dbd0877bef178c065b48bc2ff)